doc/cephadm/troubleshooting.rst

```diff
@@ -118,23 +118,47 @@ cephadm log file called ``ceph.cephadm.log`` on all monitor hosts (see
 Gathering log files
 -------------------
 
-Use journalctl to gather the log files of all daemons:
+General Procedure for Gathering Log Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. note:: By default cephadm now stores logs in journald. This means
-   that you will no longer find daemon logs in ``/var/log/ceph/``.
+Use ``journalctl`` to gather the log files of all daemons.
 
-To read the log file of one specific daemon, run::
+.. note:: Since Octopus, cephadm stores logs in journald by default. 
+          Since Octopus, daemon logs are not stored in 
+          ``/var/log/ceph/``.
```

Comment:
cephadm was introduced with octoups.

Reply:
I know, but I don't understand what you're suggesting here.

Reply:
i think what Sebastian suggests is to remove "Since Octopus". it's quite like a novel puts:

> Joe was born in 1994.
> ...
> Since 1994, Joe has a habit of blinking his eyelids when he starts to feel nervous.

readers with a bad memory would have a wrong impression that something happened to Joe in 1994. But actually, nothing actually happened. the same applies to cephadm. this is not a change introduced in octopus, cephadm was born with this behavior.

Reply:
That's clever. I get it.

I've removed "Since Octopus".

Reply:
thank you kefu!